applications/GeoMechanicsApplication/custom_strategies/strategies/geo_mechanics_newton_raphson_erosion_process_strategy.hpp

```diff
@@ -239,53 +219,42 @@ class GeoMechanicsNewtonRaphsonErosionProcessStrategy
     /// </summary>
     /// <param name="PipeElements"></param>
     /// <returns></returns>
-    int InitialiseNumActivePipeElements(std::vector<Element*> PipeElements)
+    template <typename FilteredElementsType>
+    SizeType GetNumberOfActivePipeElements(const FilteredElementsType& PipeElements)
```

Comment:
To follow the Kratos Style Guide:
```suggestion
    SizeType GetNumberOfActivePipeElements(const FilteredElementsType& rPipeElements)
```

Reply:
done